### DISCUSSION TITLE
Error in property name oauth2.ldap.activeDirectory.domain

### DISCUSSION THREAD

**Discussion by IceGrayX** [provectus/kafka-ui#4422] [General]
Hello!

I don’t know where to write correctly, so I’ll write in this type of discussion

I was dealing with the problem of authorization through Active Directory and came across an error in the name of the property: oauth2.ldap.activeDirectory.domain - the word active uses the Cyrillic alphabet c, but it should be the Latin alphabet c.

Class: com.provectus.kafka.ui.config.auth.LdapProperties
```java
@ConfigurationProperties("spring.ldap")
@Data
public class LdapProperties {

  private String urls;
  private String base;
  private String adminUser;
  private String adminPassword;
  private String userFilterSearchBase;
  private String userFilterSearchFilter;
  private String groupFilterSearchBase;
  private String groupFilterSearchFilter;
  private String groupRoleAttribute;

  @Value("${oauth2.ldap.activeDirectory:false}")
  private boolean isActiveDirectory;
  @Value("${oauth2.ldap.aсtiveDirectory.domain:@null}") //the problem is here 
  private String activeDirectoryDomain;

}
```

Please pay attention to this.
Otherwise it took some time to find the problem.
Thank you!

P.S.: Oh yes, the problem is in the master branch

**Comment by Haarolean** [comment]
Hey, thanks for the heads up. That's awkward, no idea how this happened!
Fixed it here: https://github.com/kafbat/kafka-ui/pull/252
